stable/pomerium/values.yaml

```diff
@@ -121,3 +121,9 @@ image:
 metrics:
   enabled: false
   port: 9090
+
+serviceMonitor:
+  enabled: false
+  namespace: false
```

Comment:
if you set this to false, helm will be confused when the user then set it to a string. Either leave it commented or maybe set it to the empty string? (I don't like that second option but I'm not 100% sure what's best).

Reply:
Good catch 👍 empty string is the right way, I think.